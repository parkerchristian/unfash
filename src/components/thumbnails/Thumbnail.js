import React from 'react';
import PropTypes from 'prop-types';

function Thumbnail({ imgUrl, title }) {
  const styles = {
    width: '200px',
    height: 'auto'
  };

  return (
    <>
      <h3>{title}</h3>
      <img style={styles} src={imgUrl} />
    </>
  );
}

Thumbnail.propTypes = {
  imgUrl: PropTypes.string.isRequired,
  title: PropTypes.string.isRequired
};

export default Thumbnail;
