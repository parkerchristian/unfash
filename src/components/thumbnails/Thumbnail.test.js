import React from 'react';
import Thumbnail from './Thumbnail';
import { shallow } from 'enzyme';

describe('Thumbnail component', () => {
  it('renders a thumbnail', () => {
    const imgUrl = 'abc.com';
    const title = 'Look at this stupid outfit';
    const wrapper = shallow(<Thumbnail imgUrl={imgUrl} title={title} />);
    expect(wrapper).toMatchSnapshot();
  });
});
