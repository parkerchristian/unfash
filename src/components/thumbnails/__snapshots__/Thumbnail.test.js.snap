// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thumbnail component renders a thumbnail 1`] = `
<Fragment>
  <h3>
    Look at this stupid outfit
  </h3>
  <img
    src="abc.com"
  />
</Fragment>
`;
